### DISCUSSION TITLE
Let's fix current issues and improve the documentation site! 💪

### DISCUSSION THREAD

**Discussion by evert-arias** [evert-arias/EasyButton#95] [General]
Hey everyone,

I'll be going through all the open PRs and issues in the coming weeks. I'll also be updating the documentation website with the suggestions you've all given me.

Feel free to drop your comments here.

**Comment by jsolderitsch** [comment]
The `filter.h` issue just raised it's head for me. `MegunoLink` seems broken with the recent ESP32 board updates. I removed that library and then got the `filter.h` error. Can the next release fix this?

**Reply by evert-arias** [reply]
The problem with filter.h will be fixed in the upcoming version.
